list.go

```diff
@@ -42,6 +42,10 @@ func ListStacks(root string) ([]Entry, error) {
 				return err
 			}
 
+			if !info.IsDir() {
```

Comment:
as a performance improvement I guess we also can return `filepath.SkipDir` in the line below: https://github.com/mineiros-io/terramate/blob/ff66ae70f2175ac9b5557a3d3c6f9ba6bee566fb/list.go#L68

so this function is not called again for files in this "parsed" directory.